hammett/utils/autodiscovery.py

```diff
@@ -6,26 +6,42 @@
 import inspect
 import pkgutil
 from types import ModuleType
+from typing import TYPE_CHECKING
 
 from hammett.core.screen import Screen
 
+if TYPE_CHECKING:
+    from collections.abc import Iterable
 
-def _autodiscover_screens_in_module(module: ModuleType) -> set[type[Screen]]:
+
+def _autodiscover_screens_in_module(
+    module: ModuleType,
+    exclude_screens: 'Iterable[type[Screen]]',
+) -> set[type[Screen]]:
```

Comment:
Wrap in single quotes
```suggestion
) -> 'set[type[Screen]]':
```